Resources/views/Resetting/email.txt.twig

```diff
@@ -0,0 +1,14 @@
+{% trans_default_domain 'FOSUserBundle' %}
+{% block subject %}
+Reset Password - CampaignChain
```

Comment:
The name of the application (in this case "CampaignChain") should be filled in dynamically from the [name property of the System entity](https://github.com/CampaignChain/core/blob/master/Entity/System.php#L44). Reason being that we want to allow this to be configurable at a later point, so that a user running their own in-house instance could name it "ACME Campaign Manager", for example.